src/App.js

```diff
@@ -1,14 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+// import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
+import LikeCounter from './components/LikeCounter'
 
 const App = () => {
+  const logCopy = chatMessages.map((messages)=> {return {...messages}})
+  const [messages,changeMessages] = useState(logCopy)
+  const [likes,updateLikes] = useState(0)
```

Comment:
React convention for state getters and setters would modify this to: 
```
const [likes, setLikes] = useState(0)
```
The code you have here still works! But it helps with readability because React developers will know that based on convention, anything that starts with `set` is a state setter and not a "normal" update function!